pkgs/development/libraries/aspell/dictionaries.nix

```diff
@@ -105,6 +105,26 @@ let
         homepage = "http://ftp.gnu.org/gnu/aspell/dict/0index.html";
       } // (args.meta or {});
 
+    } // lib.optionalAttrs (stdenv.isDarwin && elem language [ "is" "nb" ]) {
+      # tar: Cannot open: Illegal byte sequence
```

Comment:
Perhaps we can run tar with some locale env var instead?

Reply:
Oh, I forgot this was darwin-only and that linux already works.

Reply:
Tried setting `LC_ALL` but to no avail.